README.md

```diff
@@ -1,2 +1,13 @@
 # iexec-commons-containers
-A Java library for handling containers
+
+A Java library for handling containers.
+
+## Overview
+
+This repository is used in the [_iExec Worker_](https://github.com/iExecBlockchainComputing/iexec-worker).
+It provides an API to manage Docker containers through the [_docker-java_](https://github.com/docker-java/docker-java/)
+Java API client.
```

Comment:
Extra new line?
```suggestion
It provides an API to manage Docker containers through the [_docker-java_](https://github.com/docker-java/docker-java/) Java API client.
```

Reply:
Good question. The new line was to avoid a long line in editor.
As there is no blank line, the README is correctly rendered, see https://github.com/iExecBlockchainComputing/iexec-commons-containers/tree/bugfix/add-lombok-configuration